sandpack-react/src/templates/vue-ts.ts

```diff
@@ -0,0 +1,123 @@
+/**
```

Comment:
I was wondering, is there any reason why we still maintain `vue` instead only `vue3`? At some point in the future we can sunset it and provide only the latest version of Vue

Reply:
Yeah, we spoke internally, and let's focus on `vue3` only. This means we can remove this file and keep the original `vue` template, and on the next major release, we'll remove the `vue` template in favor of the `vue3`.